vmdb/lib/tasks/test.rake

```diff
@@ -1,6 +1,6 @@
 require_relative './evm_test_helper'
 
-if defined?(RSpec)
+if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
```

Comment:
@kbrock Why is this here?  Am I blind?  I don't see RSpec being used in this if block?